### PR TITLE
Add 75 Pixels Ortholinear Keyboard

### PR DESCRIPTION
**Pull request by sendz**
## Description

Add 75 keys Ortholinear Keyboard PCB

Screen Shot 2021-03-13 at 20 31 22

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sendyyeah/75pixels/75pixels.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+#include "75pixels.h"
+
+void eeconfig_init_kb(void) {
+    eeconfig_update_kb(0);
+    eeconfig_init_user();
+}
```
Comment (drashna) [inline keyboards/sendyyeah/75pixels/75pixels.c:22]: This isn't being changed from the default behavior, and can be removed.
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/sendyyeah/75pixels/75pixels.h [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT(\
```
Comment (drashna) [inline keyboards/sendyyeah/75pixels/75pixels.h:21]:
```suggestion
#define LAYOUT_ortho_5x15(\
```

File: keyboards/sendyyeah/75pixels/info.json [hunk 3]
```diff
@@ -0,0 +1,88 @@
+{
+    "keyboard_name": "beVi",
+    "url": "https://github.com/sendyyeah/bevi",
+    "maintainer": "sendyyeah",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
```
Comment (drashna) [inline keyboards/sendyyeah/75pixels/info.json:8]:
```suggestion
        "LAYOUT_ortho_5x15": {
```

File: keyboards/sendyyeah/75pixels/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+ */
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    QW,
+    FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [QW] = LAYOUT(
```
Comment (drashna) [inline keyboards/sendyyeah/75pixels/keymaps/default/keymap.c:24]:
```suggestion
    [QW] = LAYOUT_ortho_5x15(
```

File: keyboards/sendyyeah/75pixels/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        KC_ESC , KC_1   , KC_2   , KC_3  , KC_4   , KC_5  , KC_6  , KC_7  , KC_8   , KC_9   , KC_0   , KC_BSPC, KC_NLCK, KC_PSLS, KC_PAST, \
+        KC_TAB , KC_Q   , KC_W   , KC_E  , KC_R   , KC_T  , KC_Y  , KC_U  , KC_I   , KC_O   , KC_P   , KC_BSLS, KC_P7  , KC_P8  , KC_P9  , \
+        KC_CAPS, KC_A   , KC_S   , KC_D  , KC_F   , KC_G  , KC_H  , KC_J  , KC_K   , KC_L   , KC_SCLN, KC_ENT , KC_P4  , KC_P5  , KC_P6  , \
+        KC_LSFT, KC_Z   , KC_X   , KC_C  , KC_V   , KC_B  , KC_N  , KC_M  , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, KC_P1  , KC_P2  , KC_P3  , \
+        KC_LCTL, KC_LGUI, KC_LALT, MO(FN), _______, KC_SPC, KC_SPC, TO(FN), KC_RALT, KC_RGUI, KC_MENU, KC_RCTL, KC_P0  , KC_PDOT, KC_ENT
+    ),
+    [FN] = LAYOUT(
```
Comment (drashna) [inline keyboards/sendyyeah/75pixels/keymaps/default/keymap.c:31]:
```suggestion
    [FN] = LAYOUT_ortho_5x15(
```

File: keyboards/sendyyeah/75pixels/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+ */
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    QW,
+    FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [QW] = LAYOUT(
```
Comment (drashna) [inline keyboards/sendyyeah/75pixels/keymaps/via/keymap.c:24]:
```suggestion
    [QW] = LAYOUT_ortho_5x15(
```

File: keyboards/sendyyeah/75pixels/rules.mk [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (drashna) [inline keyboards/sendyyeah/75pixels/rules.mk:22]:
```suggestion
AUDIO_ENABLE = no           # Audio output

LAYOUTS = ortho_5x15
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/sendyyeah/75pixels/config.h [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x5359
+#define PRODUCT_ID      0x3735
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    sendyyeah
+#define PRODUCT         "75 Pixels"
```
Comment (fauxpark) [inline keyboards/sendyyeah/75pixels/config.h:26]:
```suggestion
#define PRODUCT         75 Pixels
```

File: keyboards/sendyyeah/75pixels/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+        KC_TAB , KC_Q   , KC_W   , KC_E  , KC_R   , KC_T  , KC_Y  , KC_U  , KC_I   , KC_O   , KC_P   , KC_BSLS, KC_P7  , KC_P8  , KC_P9  , \
+        KC_CAPS, KC_A   , KC_S   , KC_D  , KC_F   , KC_G  , KC_H  , KC_J  , KC_K   , KC_L   , KC_SCLN, KC_ENT , KC_P4  , KC_P5  , KC_P6  , \
+        KC_LSFT, KC_Z   , KC_X   , KC_C  , KC_V   , KC_B  , KC_N  , KC_M  , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, KC_P1  , KC_P2  , KC_P3  , \
+        KC_LCTL, KC_LGUI, KC_LALT, MO(FN), _______, KC_SPC, KC_SPC, TO(FN), KC_RALT, KC_RGUI, KC_MENU, KC_RCTL, KC_P0  , KC_PDOT, KC_ENT
+    ),
+    [FN] = LAYOUT_ortho_5x15(
+        _______,  KC_F1 ,  KC_F2 ,  KC_F3 ,  KC_F4 ,  KC_F5 ,  KC_F6 ,  KC_F7 ,  KC_F8 ,  KC_F9 , KC_F10 , KC_DEL , _______, _______, _______, \
+        _______, KC_F11 , KC_F12 , _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
```
Comment (fauxpark) [inline keyboards/sendyyeah/75pixels/keymaps/default/keymap.c:35]:
```suggestion
        KC_ESC , KC_1   , KC_2   , KC_3  , KC_4   , KC_5  , KC_6  , KC_7  , KC_8   , KC_9   , KC_0   , KC_BSPC, KC_NLCK, KC_PSLS, KC_PAST,
        KC_TAB , KC_Q   , KC_W   , KC_E  , KC_R   , KC_T  , KC_Y  , KC_U  , KC_I   , KC_O   , KC_P   , KC_BSLS, KC_P7  , KC_P8  , KC_P9  ,
        KC_CAPS, KC_A   , KC_S   , KC_D  , KC_F   , KC_G  , KC_H  , KC_J  , KC_K   , KC_L   , KC_SCLN, KC_ENT , KC_P4  , KC_P5  , KC_P6  ,
        KC_LSFT, KC_Z   , KC_X   , KC_C  , KC_V   , KC_B  , KC_N  , KC_M  , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, KC_P1  , KC_P2  , KC_P3  ,
        KC_LCTL, KC_LGUI, KC_LALT, MO(FN), _______, KC_SPC, KC_SPC, TO(FN), KC_RALT, KC_RGUI, KC_MENU, KC_RCTL, KC_P0  , KC_PDOT, KC_ENT
    ),
    [FN] = LAYOUT_ortho_5x15(
        _______,  KC_F1 ,  KC_F2 ,  KC_F3 ,  KC_F4 ,  KC_F5 ,  KC_F6 ,  KC_F7 ,  KC_F8 ,  KC_F9 , KC_F10 , KC_DEL , _______, _______, _______,
        _______, KC_F11 , KC_F12 , _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
```

File: keyboards/sendyyeah/75pixels/keymaps/via/keymap.c [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+        KC_TAB , KC_Q   , KC_W   , KC_E  , KC_R   , KC_T  , KC_Y  , KC_U  , KC_I   , KC_O   , KC_P   , KC_BSLS, KC_P7  , KC_P8  , KC_P9  , \
+        KC_CAPS, KC_A   , KC_S   , KC_D  , KC_F   , KC_G  , KC_H  , KC_J  , KC_K   , KC_L   , KC_SCLN, KC_ENT , KC_P4  , KC_P5  , KC_P6  , \
+        KC_LSFT, KC_Z   , KC_X   , KC_C  , KC_V   , KC_B  , KC_N  , KC_M  , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, KC_P1  , KC_P2  , KC_P3  , \
+        KC_LCTL, KC_LGUI, KC_LALT, MO(FN), _______, KC_SPC, KC_SPC, TO(FN), KC_RALT, KC_RGUI, KC_MENU, KC_RCTL, KC_P0  , KC_PDOT, KC_ENT
+    ),
+    [FN] = LAYOUT(
+        _______,  KC_F1 ,  KC_F2 ,  KC_F3 ,  KC_F4 ,  KC_F5 ,  KC_F6 ,  KC_F7 ,  KC_F8 ,  KC_F9 , KC_F10 , KC_DEL , _______, _______, _______, \
+        _______, KC_F11 , KC_F12 , _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
```
Comment (fauxpark) [inline keyboards/sendyyeah/75pixels/keymaps/via/keymap.c:35]:
```suggestion
        KC_ESC , KC_1   , KC_2   , KC_3  , KC_4   , KC_5  , KC_6  , KC_7  , KC_8   , KC_9   , KC_0   , KC_BSPC, KC_NLCK, KC_PSLS, KC_PAST,
        KC_TAB , KC_Q   , KC_W   , KC_E  , KC_R   , KC_T  , KC_Y  , KC_U  , KC_I   , KC_O   , KC_P   , KC_BSLS, KC_P7  , KC_P8  , KC_P9  ,
        KC_CAPS, KC_A   , KC_S   , KC_D  , KC_F   , KC_G  , KC_H  , KC_J  , KC_K   , KC_L   , KC_SCLN, KC_ENT , KC_P4  , KC_P5  , KC_P6  ,
        KC_LSFT, KC_Z   , KC_X   , KC_C  , KC_V   , KC_B  , KC_N  , KC_M  , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, KC_P1  , KC_P2  , KC_P3  ,
        KC_LCTL, KC_LGUI, KC_LALT, MO(FN), _______, KC_SPC, KC_SPC, TO(FN), KC_RALT, KC_RGUI, KC_MENU, KC_RCTL, KC_P0  , KC_PDOT, KC_ENT
    ),
    [FN] = LAYOUT(
        _______,  KC_F1 ,  KC_F2 ,  KC_F3 ,  KC_F4 ,  KC_F5 ,  KC_F6 ,  KC_F7 ,  KC_F8 ,  KC_F9 , KC_F10 , KC_DEL , _______, _______, _______,
        _______, KC_F11 , KC_F12 , _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
```

File: keyboards/sendyyeah/75pixels/keymaps/via/keymap.c [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+
```
Comment (fauxpark) [inline keyboards/sendyyeah/75pixels/keymaps/via/keymap.c:37]: Need two extra empty layers here.

File: keyboards/sendyyeah/75pixels/readme.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+
+* Keyboard Layout: [KLE](http://www.keyboard-layout-editor.com/#/gists/3d64c8d06f0b1a8cf5b58797a2dae8dc)
+* Keyboard Maintainer: [SENDYYeah](https://github.com/sendyyeah)
+* Hardware Availability: [Tokopedia/Sell Stuffs](https://tokopedia.com/sell-stuffs) (Indonesia only) or any electronic part store.
+
+Make example for this keyboard (after setting up your build environment):
+
+    `make sendyyeah/75pixels:default`
+    or
+    `qmk compile -kb sendyyeah/75pixels -km default`
```
Comment (fauxpark) [inline keyboards/sendyyeah/75pixels/readme.md:15]:
```suggestion
    make sendyyeah/75pixels:default
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/sendyyeah/75pixels/keymaps/via/keymap.c (hunk 11 above)
Comment (sendz) [reply keyboards/sendyyeah/75pixels/keymaps/via/keymap.c:37]: added

File: keyboards/sendyyeah/75pixels/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+        KC_TAB , KC_Q   , KC_W   , KC_E  , KC_R   , KC_T  , KC_Y  , KC_U  , KC_I   , KC_O   , KC_P   , KC_BSLS, KC_P7  , KC_P8  , KC_P9  ,
+        KC_CAPS, KC_A   , KC_S   , KC_D  , KC_F   , KC_G  , KC_H  , KC_J  , KC_K   , KC_L   , KC_SCLN, KC_ENT , KC_P4  , KC_P5  , KC_P6  ,
+        KC_LSFT, KC_Z   , KC_X   , KC_C  , KC_V   , KC_B  , KC_N  , KC_M  , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, KC_P1  , KC_P2  , KC_P3  ,
+        KC_LCTL, KC_LGUI, KC_LALT, MO(FN), _______, KC_SPC, KC_SPC, TO(FN), KC_RALT, KC_RGUI, KC_MENU, KC_RCTL, KC_P0  , KC_PDOT, KC_ENT
+    ),
+    [FN] = LAYOUT_ortho_5x15(
+        _______,  KC_F1 ,  KC_F2 ,  KC_F3 ,  KC_F4 ,  KC_F5 ,  KC_F6 ,  KC_F7 ,  KC_F8 ,  KC_F9 , KC_F10 , KC_DEL , _______, _______, _______,
+        _______, KC_F11 , KC_F12 , _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
```
Comment (fauxpark) [inline keyboards/sendyyeah/75pixels/keymaps/default/keymap.c:35]:
```suggestion
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
```

File: keyboards/sendyyeah/75pixels/readme.md [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make sendyyeah/75pixels:default
+
+How to reset and enter bootloader:
+    - Press reset button on the PCB (twice if the ProMicro is fresh)
+    - Flash with QMK Toolbox or any command line you like
+
+    See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Comment (fauxpark) [inline keyboards/sendyyeah/75pixels/readme.md:19]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/sendyyeah/75pixels/keymaps/via/keymap.c (hunk 11 above)
Comment (drashna) [reply keyboards/sendyyeah/75pixels/keymaps/via/keymap.c:37]: Still missing, it looks like.  Did you push the changes?

Comment (sendz) [reply keyboards/sendyyeah/75pixels/keymaps/via/keymap.c:55]: ah yes, you're right, I ignored the conflict message when push last day, thanks

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**